package.json

```diff
@@ -56,6 +56,7 @@
     "express": "4.13.3",
     "filesize": "3.2.1",
     "flux": "2.1.1",
+    "get-video-id": "1.1.0",
```

Comment:
👍 

I think you'll also need to do `make shrinkwrap` to update the shrinkwrap file.